pupil_src/capture/pupil_detectors/detector_3d.pyx

```diff
@@ -63,26 +63,27 @@ cdef class Detector_3D:
         self.detectProperties3D = settings['3D_Settings'] if settings else {}
 
         if not self.detectProperties2D:
-            self.detectProperties2D["coarse_detection"] = True
-            self.detectProperties2D["coarse_filter_min"] = 150
-            self.detectProperties2D["coarse_filter_max"] = 400
-            self.detectProperties2D["intensity_range"] = 17
-            self.detectProperties2D["blur_size"] = 3
-            self.detectProperties2D["canny_treshold"] = 200
-            self.detectProperties2D["canny_ration"] = 3
-            self.detectProperties2D["canny_aperture"] = 5
-            self.detectProperties2D["pupil_size_max"] = 150
-            self.detectProperties2D["pupil_size_min"] = 40
-            self.detectProperties2D["strong_perimeter_ratio_range_min"] = 0.8
-            self.detectProperties2D["strong_perimeter_ratio_range_max"] = 1.1
-            self.detectProperties2D["strong_area_ratio_range_min"] = 0.6
-            self.detectProperties2D["strong_area_ratio_range_max"] = 1.1
-            self.detectProperties2D["contour_size_min"] = 5
-            self.detectProperties2D["ellipse_roundness_ratio"] = 0.1
-            self.detectProperties2D["initial_ellipse_fit_treshhold"] = 1.8
-            self.detectProperties2D["final_perimeter_ratio_range_min"] = 0.6
-            self.detectProperties2D["final_perimeter_ratio_range_max"] = 1.2
-            self.detectProperties2D["ellipse_true_support_min_dist"] = 2.5
+            self.detectProperties["coarse_detection"] = True
```

Comment:
@marc-tonsen - all of these lines L66:85 should be `self.detectProperties2D` right?